lighthouse-core/test/lib/url-shim-test.js

```diff
@@ -114,6 +114,17 @@ describe('URL Shim', () => {
     });
   });
 
+  describe('getRootDomain', () => {
+    it('returns the correct rootDomain', () => {
+      assert.equal(URL.getRootDomain('example.com'), 'example.com');
```

Comment:
the input is expected to be a full url, right? with `http://`, etc.. can we use those?

Reply:
hostname is used, if I didn't than I had to create an url object again. I could be smart about it and accept a string or URL object. (this could be a function that we use everywhere in the class so we only have to createn URL once.

```
let hostname;
if (url instanceof URL) {
  hostname = url.hostname;
} else {
  const urlObj = new URL(url);
  hostname = urlObj.hostname;
}
```

Reply:
sg.